### ISSUE TITLE
Optional blue line request

### ISSUE THREAD

**Issue by JayJay1968** [Banjobeni/DVDispatcherMod#8]
I wish it was possible to disable the blue line pointing to the given track. I find it rather distracting. Maybe you could make it an optional setting?

**Comment by T3M8029** [comment]
Added this as a setting in a [pull request](https://github.com/Banjobeni/DVDispatcherMod/pull/7), you can download the file from there for now and test if everything´s ok and potentially give feedback.

**Comment by JayJay1968** [comment]
Hi,

Yes, perfect!
If I should nitpick, the text box size could be reduced by 50%. But that's
my personal opinion. People might like how it is right now.

Thank you!

Best regards,
Jonny Jorgensen

**Comment by CrankyAnt** [comment]
> Added this as a setting in a [pull request](https://github.com/Banjobeni/DVDispatcherMod/pull/7), you can download the file from there for now and test if everything´s ok and potentially give feedback.

Thanks, thats much better. Now I noticed that some job descriptions (accepted and non-accepted) persist in the messagebox on the top left until you reload the game. On accepted jobs it has no influence to complete them, non-accepted jobs can get stuck there even when you just looked at them. 
It might be better to create a seperate issue for this in the original mod, but as Im using your pull and it hasnt been merged yet I'm not sure what to do. 

Attached detailed log extracted from UMM
[Player.log](https://github.com/user-attachments/files/19672872/Player.log)

**Comment by T3M8029** [comment]
I also noticed this sometimes, but haven´t yet found the reason behund it, I believe it is also an issue in the master branch introduced with #6  because of compatibility with other mods (#4 ), and as mentioned there it warrants a deeper look at some point (no time now probably tho). For now I can "fix" this by putting a refresh button in the settings, that clears all current boxes, so hope that will suffice for now. Also don´t have the debug job logging option enabled when normally playing, it can slow down the game since it prints a lot of stuff to the log. (Also also if you´d be up for testing out a build of Persistant Jobs it would be great, you can find it [here](https://github.com/Banjobeni/DerailValley-PersistentJobs/pull/23) ).

**Comment by T3M8029** [comment]
It should be somewhat mitigated here: [Prebuilt Dll](https://github.com/user-attachments/files/19719935/DVDispatcherMod.zip)
 , so you can update to this and see if it´s alright.

**Comment by CrankyAnt** [comment]
> It should be somewhat mitigated [here](https://github.com/user-attachments/files/19674718/DVDispatcherMod.zip) , so you can update to this and see if it´s alright.

Thanks, a refresh or duration will definitely help enough. 
And testing the PJ build, of course on it! Is it the prebuild dll mentioned in the post from december 2024?

good tip on the debug log, its convenient having it on though when reporting an error but have some CTD's now and then so will see if turning it off again helps with that.

**Comment by T3M8029** [comment]
> And testing the PJ build, of course on it! Is it the prebuild dll mentioned in the post from december 2024?

Yeah, I replace the file there when changing anything, but I can put it also here.

[PersistentJobsMod.zip](https://github.com/user-attachments/files/19695118/PersistentJobsMod.zip)
